flask_request_validator/rules.py

```diff
@@ -159,42 +136,23 @@ def __init__(self, value: Union[int, float], include_boundary: bool = True) -> N
         self._value = value
         self._include_boundary = include_boundary
 
-    @overrides(AbstractRule)
-    def validate(self, value: Union[int, float]) -> Tuple[Union[int, float], List[str]]:
+    def validate(self, value: Union[int, float]) -> List[str]:
         errors = []
 
         if value < self._value and self._include_boundary:
             errors.append(f'smaller then allowed: {value} is not >= {self._value}')
         elif value <= self._value and not self._include_boundary:
             errors.append(f'smaller then allowed: {value} is not > {self._value}')
 
-        return value, errors
-
-
-class IsDatetimeIsoFormat(AbstractRule):
```

Comment:
@LostInDarkMath is not a `Rule`. Rules works with converted values. It should be a new `Param`